src/components/sendV2/send.test.js

```diff
@@ -100,7 +100,17 @@ describe('Form', () => {
     wrapper = mount(<Send {...props} />, options);
   });
 
-  it('should render properly', () => {
+  it('should render properly getting data from URL', () => {
+    expect(wrapper).toContainMatchingElement('.send-box');
+    expect(wrapper).toContainMatchingElement('MultiStep');
+    expect(wrapper).toContainMatchingElement('Form');
+  });
+
+  it('should render properly without getting data from URL', () => {
+    props.history.location.path = '';
+    props.history.location.search = '';
```

Comment:
Avoid mutating test variables that are used in all tests, like this it will affect all other tasks that come after this one, it's better to create a new object.